client/landing/jetpack-cloud/components/backup-activity-log/date-range-selector.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { Moment } from 'moment';
+import { useTranslate } from 'i18n-calypso';
+import React, { FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import DateRange from 'components/date-range';
+
+interface Props {
+	newestDate: Moment;
+	oldestDate: Moment;
+	onDateCommit: ( selectedStartDate: Moment | null, selectedEndDate: Moment | null ) => void;
+	selectedEndDate: Moment | null;
+	selectedStartDate: Moment | null;
+}
+
+const BackupsDateRangeSelector: FunctionComponent< Props > = ( {
+	oldestDate,
+	newestDate,
+	onDateCommit,
+	selectedEndDate,
+	selectedStartDate,
+} ) => {
+	const translate = useTranslate();
+
+	const renderTrigger = ( { buttonRef, onTriggerClick } ) => (
+		<Button ref={ buttonRef } onClick={ onTriggerClick }>
+			{ translate( 'Date range' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **126** times:
`translate( 'Date range:' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-cloud-basic-activity-log)